dw-date-input.js

```diff
@@ -297,4 +298,6 @@ export class DwDateInput extends DwFormElement(LitElement) {
   }
 }
 
-window.customElements.define("dw-date-input", DwDateInput);
+if (!isElementAlreadyRegistered("dw-date-input")) {
```

Comment:
if element is already registered, add a WARN log. 

lit: 'dw-date-input' is already registered, so registration skipped.